src/particles/media/documentation.stories.mdx

```diff
@@ -44,6 +44,20 @@ export const Template = (args) => (
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story name='Video Without Extension'>
+    <Media source='https://gateway.pinata.cloud/ipfs/QmZ3yCYX2c6Z2BNSTyx1zuBoPUJTB8dyE74zwngdweaFV6' />
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name='image Without Extension'>
```

Comment:
```suggestion
  <Story name='Image Without Extension'>
```

Reply:
is this one an image or an iframe? If iframe pls update the `name` and add one with an image